pkgs/applications/editors/ne/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, ncurses, texinfo, tetex, perl, ghostscript }:
+
+
+stdenv.mkDerivation {
+  name = "ne-3.0.1";
+  version = "3.0.1";
+  src = fetchurl {
+    url = https://github.com/vigna/ne/archive/3.0.1.tar.gz;
+    md5 = "ec3dc82d10c271e95ecde630538f9f19";
+  };
+  buildInputs = [ ncurses tetex texinfo perl ghostscript ];
+  preBuild = ''
+    substituteInPlace src/makefile --replace "CC=c99" "cc=gcc"
+    substituteInPlace src/makefile --replace "-lcurses" "-lncurses"
+    substituteInPlace makefile --replace "./version.pl" "perl version.pl"
+    cd doc && make && cd ..
+    cd src && make && cd ..
+    make PREFIX=$out install
```

Comment:
if I just let the genericBuilder run make PREFIX=$out install it doesn't throw errors but the files are not there after it's done for some reason, while if I write the command myself it works. I'm not sure why but I'm afraid I wasn't able to figure it out.